docs/README.md

```diff
@@ -1,26 +1,26 @@
 # React-Bootstrap Documentation Website
 
-This website is single page app built on
+This website is a single page app built on
 [React](http://facebook.github.io/react/), with styles and structure taken from
 the [Bootstrap](http://getbootstrap.com/) docs website.  The app is statically
-generated to HTML via node and then hosted it by pushing HTML to [GitHub
+generated to HTML via node and then hosted by pushing HTML to [GitHub
```

Comment:
capitalize "Node"?

Reply:
Yes, I believe so. According to the website it is spelled "Node.js". Node, being shorthand should probably still conform to the uppercase proper name style choice. Good call!